src/com/company/OOP/phone/Main.java

```diff
@@ -0,0 +1,7 @@
+package com.company.OOP.phone;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
The task requirements were the next:
To test the program, in main() method, create three balls using getRandomBall() method from Lottery class. And print information about them in the console.